t/028-req-header.t

```diff
@@ -1622,3 +1622,24 @@ ok
 --- no_error_log
 [error]
 --- no_check_leak
+
+
+
+=== TEST 54: for bad requests causing segfaults when setting & getting multi-value headers
```

Comment:
Totally optional but I'd consider changing the test description to something like `TEST 54: for multi-value header manipulation during bad requests (bad method letter case)` in order to follow the description format of the other tests. I realized that TEST 54 is based on TEST 52, but it wasn't immediately obvious that RFC 2616 5.1.1 stipulates case-sensitive method names.